### PR TITLE
completed the solution of homework in lesson 2 in file main.js

### PR DESCRIPTION
**Pull request by Irina-pi**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+/*task 7*/
+let test = 1;
+
+test++ /*2*/
+test = test + '1' /*21*/
+
+console.log(test);
+
+test = test - 1 /*0*/
```
Comment (5tronciy) [inline main.js]: Wrong comment. Check it in the console.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+console.log(test);
+
+test = true; /*true*/
```
Comment (5tronciy) [inline main.js]: It doesn't assign the task. You should convert the value of the variable `test` to boolean.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+console.log(test);
+
+/*task 8*/
+let arr = [2, 3, 5, 8];
+let result = 1;
+let i = 0;
+
+for (; i < arr.length; i++) {
```
Comment (5tronciy) [inline main.js]: Why not to move `let i = 0` into the place of the missed argument here?

File: package.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
   },
   "homepage": "https://github.com/Irina-pi/if-js#readme",
   "devDependencies": {
-    "eslint": "^8.23.0",
+    "eslint": "8.22.0",
```
Comment (5tronciy) [inline package.json:33]: What wrong with the previous?

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Remove the folder `.idea`. It should be ignored with the file `.gitignore`.

**Comment by 5tronciy** [comment]
There is empty folder `if-js`. Also remove it.

File: main.js (hunk 1 above)
Comment (Irina-pi) [reply main.js]: I understood why it.

Comment (Irina-pi) [reply main.js]: I have reassigned the value of the variable.

File: main.js (hunk 2 above)
Comment (Irina-pi) [reply main.js]: Ok. I understood and corrected it.

File: main.js (hunk 3 above)
Comment (Irina-pi) [reply main.js]: I will do so. made different choices.

File: package.json (hunk 4 above)
Comment (Irina-pi) [reply package.json:33]: when I created a new branch, WebShtorm offered to automatically install ESlintrc and apparently this version.

File: main.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+test = test - 1; /*20*/
+
+console.log(test);
+
+if (test === 20) {
+    console.log(true);
+} else {
+    console.log(false);
+}
```
Comment (5tronciy) [inline main.js]: What is it for?
Google the ways to convert the value of variable to a boolean.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Where is `.gitignore`?
And the empty folder `if-js` still exists.

File: main.js (hunk 5 above)
Comment (Irina-pi) [reply main.js]: I think I  understood all and corrected it.

File: main.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+test++; /*2*/
+test = test + '1'; /*21*/
+
+console.log(test);
+
+test = test - 1; /*20*/
+
+console.log(test);
+
+const bool = Boolean(test);
```
Comment (5tronciy) [inline main.js]: It's better but doesn't change the variable `test` as required. It could be similar to the previous operations.

File: main.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+console.log(bool);
+
+/*task 8*/
+let arr = [2, 3, 5, 8];
+let result = 1;
+
+for (i = 0; i < arr.length; i++) {
```
Comment (5tronciy) [inline main.js]: You create the variable `i` in Global scope declaring it without `let`.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
You are moving in the right direction. You need to move on a little bit more)

File: main.js (hunk 7 above)
Comment (Irina-pi) [reply main.js]: Yes, exactly!

Comment (Irina-pi) [reply main.js]: I corrected it.

File: main.js (hunk 6 above)
Comment (Irina-pi) [reply main.js]: ok.

**Review by 5tronciy — APPROVED** [review]
Looks clear now - good work!
Approve.
You may merge this PR now.

**Pull request merged**